docs/user/dql/functions.rst

```diff
@@ -1762,6 +1762,28 @@ Example::
     | 2                           |
     +-----------------------------+
 
+MINUTE_OF_DAY
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: minute_of_day(time) returns the minute value for time within a 24 hour day, in the range 0 to 1439.
+
+Argument type: STRING/TIME/DATETIME/TIMESTAMP
+
+Return type: INTEGER
```

Comment:
Get ready - you could be asked why not `SHORT`
Hint: it uses same amount of memory as `INT`